app/nexus/internal/initialization/initialization.go

```diff
@@ -31,8 +30,10 @@ import (
 func Initialize(source *workloadapi.X509Source) {
 	requireBootstrapping := env.BackendStoreType() == env.Sqlite
 	if requireBootstrapping {
-		bootstrap(source)
+		// Try bootstrapping in a loop.
+		go bootstrap(source)
```

Comment:
Need to run this in a goroutine, otherwise the main SPIKE Nexus REST server would never start and it would be impossible to `spike operator recover` because there won't be any endpoint to listen to it.

And alternative option could be to make the `recover` functionality as part of the Nexus binary (instead of the Pilot binary), but I didn't want to give shell access to Nexus. -- an mTLS RESTful API is (arguably) more secure as we can guard it with SVIDs.